tcadmin/diff.py

```diff
@@ -95,9 +118,7 @@ def contextualize(rangeInfo):
                 return match.group(1)
         return ""
 
-    lines = patiencediff.unified_diff(
-        left, right, lineterm="", fromfile="current", tofile="generated", n=context
-    )
+    lines = fast_diff(left, right, context)
```

Comment:
If we're concerned about supporting systems without a `diff`, we could either allow someone to fall back to `patiencediff` or allow for a commandline option to specify that. I'm not sure if there are any systems we want to support without a `diff` though.